infra/event-sync/src/androidTest/java/com/simprints/infra/eventsync/remote/EventRemoteDataSourceImplAndroidTest.kt

```diff
@@ -1,8 +1,10 @@
+// TODO fix
```

Comment:
Will we ever? :D

Reply:
It is more likely do be deleted, but not in scope of this PR :D